cookbook/mirabella-genio-bulb.rst

```diff
@@ -0,0 +1,175 @@
+      # https://esphome.io/components/esphome#esphome-on-boot
+      on_boot:
+        priority: 100 # Highest priority, ensures light turns on without delay.
+        then:
+          - light.turn_on: light
+
+    wifi:
+      ssid: 'WIFI'
+      password: 'WIFIPASS'
+
+    logger:
+
+    web_server:
+      port: 80
+
+    api:
+
+    ota:
+
+    light:
+      - platform: monochromatic
+        name: "Mirabella Genio Smart Bulb"
+        id: light
+        output: output_component1
+
+    output:
+      - platform: esp8266_pwm
+        id: output_component1
+        pin: GPIO13
+
+Tunable Warm/Cool White Bulbs
+-----------------------------
+
+.. code-block:: yaml
+
+    esphome:
+      name: mirabella_genio_cwww_1
+      platform: ESP8266
+      board: esp01_1m
+
+      # Ensure the light turns on by default if the physical switch is actuated.
+      # https://esphome.io/components/esphome#esphome-on-boot
+      on_boot:
+        priority: 100 # Highest priority, ensures light turns on without delay.
+        then:
+          - light.turn_on: light
+      
+    wifi:
+      ssid: 'WIFI'
+      password: 'WIFIPASS'
+
+    logger:
+
+    web_server:
+      port: 80
+
+    api:
+
+    ota:
+
+    output:
+      - platform: esp8266_pwm
+        id: output_warm_white
+        pin: GPIO14
+      - platform: esp8266_pwm
+        id: output_daylight
+        pin: GPIO5
+
+    light:
+      - platform: cwww
+        name: "Mirabella Genio Smart Bulb"
+        id: light
+        cold_white: output_daylight
+        warm_white: output_warm_white
+        cold_white_color_temperature: 6500 K
+        warm_white_color_temperature: 2700 K
+
+RGBW Colour Bulbs
+-----------------
+
+.. code-block:: yaml
+
+    esphome:
+      name: rgbw_e27_01
+      platform: ESP8266
+      board: esp01_1m
+
+      # Ensure the light turns on by default if the physical switch is actuated.
+      # https://esphome.io/components/esphome#esphome-on-boot
+      on_boot:
+        priority: 100 # Highest priority, ensures light turns on without delay.
+        then:
+          - light.turn_on: light
+
+    wifi:
+      ssid: 'WIFI'
+      password: 'WIFIPASS'
+
+    logger:
+
+    web_server:
+      port: 80
+
+    api:
+
+    ota:
+
+    output:
+      - platform: esp8266_pwm
+        id: output_red
+        pin: GPIO4
+      - platform: esp8266_pwm
+        id: output_green
+        pin: GPIO12
+      - platform: esp8266_pwm
+        id: output_blue
+        pin: GPIO14
+      - platform: esp8266_pwm
+        id: output_white
+        pin: GPIO5
+
+    light:
+      - platform: rgbw
+        name: "rgbw_e27_01"
+        id: light
+        red: output_red
+        green: output_green
+        blue: output_blue
+        white: output_white
+
+
+Addding to Home Assistant
+-------------------------
+
+You can now add your bulb to home assistant via the configurations page, look for 'ESPHome' under the Integrations option and click 'Configure'.
+
```

Comment:
If possible, add a screenshot here of what it looks like in HA - best with the RGB version and with the color picker open.